src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/forge_of_souls.cpp

```diff
@@ -29,6 +29,9 @@ enum Yells
 
     SAY_JAINA_OUTRO                             = 0,
     SAY_SYLVANAS_OUTRO                          = 0,
+
+    GOSSIP_JAINA_INTRO                          = 10943,
+    GOSSIP_SYLVANAS_INTRO                       = 10971,
```

Comment:
don't add them in this enum, it is used for things they say

Reply:
Should I just add the numbers? I wanted to put them as constants just in case. But I didn't know whether to put the values directly or where.

Reply:
Thank you, I just added the suggested correction.